examples/objectdetection/ios/Podfile.lock

```diff
@@ -1409,7 +1409,7 @@ SPEC CHECKSUMS:
   ReactNativeMediaPipe: 92850eb7623b4bb0d6e8fe90c5e5781b42ccba00
   SocketRocket: f32cd54efbe0f095c4d7594881e52619cfe80b17
   VisionCamera: 6c8ca7d04fc62fafccd8f8ad584e96f5303671f5
-  Yoga: d17d2cc8105eed528474683b42e2ea310e1daf61
+  Yoga: 805bf71192903b20fc14babe48080582fee65a80
```

Comment:
This was what happened when I did a pod install.  I was expecting other changes, since this was one of the steps for installing react-navigation, I won't commit this PR until I get things running on the mac...

Reply:
This is now more what I'd expect (and the change to Yoga disappeared).